web/modules/custom/joinup_licence/joinup_licence.info.yml

```diff
@@ -5,32 +5,31 @@ core: 8.x
 package: Joinup
 
 dependencies:
-  - 'field_group:field_group'
+  - 'drupal:taxonomy'
   - 'joinup:joinup_validation'
-  - 'smart_trim:smart_trim'
   - 'spdx:spdx'
 
 config_devel:
   install:
     - core.base_field_override.rdf_entity.licence.label
-    - core.entity_form_display.rdf_entity.licence.default
-    - core.entity_view_display.rdf_entity.licence.default
     - core.entity_view_mode.rdf_entity.view_mode_search_list_item
     - field.field.rdf_entity.licence.field_licence_deprecated
-    - field.field.rdf_entity.licence.field_licence_description
-    - field.field.rdf_entity.licence.field_licence_legal_type
     - field.field.rdf_entity.licence.field_licence_spdx_licence
-    - field.field.rdf_entity.licence.field_licence_type
     - field.storage.rdf_entity.field_licence_deprecated
     - field.storage.rdf_entity.field_licence_description
-    - field.storage.rdf_entity.field_licence_legal_type
     - field.storage.rdf_entity.field_licence_spdx_licence
-    - field.storage.rdf_entity.field_licence_type
     - filter.format.licence
     - rdf_entity.rdfentity.licence
     - sparql_entity_storage.mapping.rdf_entity.licence
-    - sparql_entity_storage.mapping.taxonomy_term.licence_type
-    - taxonomy.vocabulary.licence_type
   optional:
+    - core.entity_form_display.rdf_entity.licence.default
+    - core.entity_view_display.rdf_entity.licence.default
     - core.entity_view_display.rdf_entity.licence.view_mode_search_list_item
     - core.entity_view_display.rdf_entity.licence.view_mode_tile
+    - field.field.rdf_entity.licence.field_licence_description
+    - field.field.rdf_entity.licence.field_licence_legal_type
+    - field.field.rdf_entity.licence.field_licence_type
+    - field.storage.rdf_entity.field_licence_legal_type
+    - field.storage.rdf_entity.field_licence_type
+    - sparql_entity_storage.mapping.taxonomy_term.licence_type
+    - taxonomy.vocabulary.licence_type
```

Comment:
Joinup as a project doesn't use these config folders. Joinup is installed using the config from `config/sync`. The only moment the module config is ever used is during KernelTests.

The canonical config is in `config/sync`, and since we have no way to sync the 2 config stores, we have to consider the config in our modules as potentially outdated. So this means that all KernelTests should also be considered as useless. They should be converted to either ExistingSite tests or Behat tests. One of these cases is already being fixed in #2100.

The end goal is to REMOVE all config from all modules, but this is at the moment not possible because it would break the remaining KernelTests. So this is a 2 step approach: first we move config to optional, then finally we can remove it.